packages/application-shell/src/components/project-container/project-container.tsx

```diff
@@ -37,7 +37,7 @@ type QueryParams = {
 };
 type TProjectContainerProps = {
   user: TFetchLoggedInUserQuery['user'];
-  environment: TProviderProps<{ enableSignUp?: boolean }>['environment'];
```

Comment:
I checked in our internal apps and we don't seem to set this variable anymore in the app config. It should be safe to remove it.

Reply:
It's a feature flag not an environment config now.